master/buildbot/util/httpclientservice.py

```diff
@@ -63,6 +65,17 @@ def json(self):
     def code(self):
         return self._res.status_code
 
+    @property
+    def headers(self):
+        if self._headers is None:
```

Comment:
according to codecov, this method is not tested. I think it is not actually needed/used

Reply:
It will be needed for my other PR. Decided to include I here because its also about headers. Can pull it out for now if you prefer.